File: 1295. Find Numbers with Even Number of Digits/1295.c
int findNumbers(int* nums, int numsSize){
    int count = 0;
    for(int i=0; i<numsSize; i++){
        if (nums[i]!=0){
            int tmp = 0;
            while(nums[i]!=0){
                nums[i] = nums[i]/10;
                tmp++;
            }
            if (tmp%2 == 0){
                count ++;
            }
        }
    }
    return count;
}

File: 1295. Find Numbers with Even Number of Digits/_1295.py
class Solution:
    def findNumbers(self, nums) -> int:
        count = 0
        for i in nums:
            if len(str(i))%2 == 0:
                count += 1
        return count

    def findNumbers1(self, nums) -> int:
        return sum(len(str(n)) % 2 == 0 for n in nums) 